source/guide_wordpress-bedrock.rst

```diff
@@ -0,0 +1,248 @@
+  - Installing roots/wordpress-core-installer (1.1.0): Downloading (100%)
+  - Installing composer/installers (v1.8.0): Downloading (100%)
+  - Installing oscarotero/env (v1.2.0): Downloading (100%)
+  - Installing roots/wordpress (5.3.2): Downloading (100%)
+  - Installing roots/wp-config (1.0.0): Downloading (100%)
+  - Installing roots/wp-password-bcrypt (1.0.0): Downloading (100%)
+  - Installing symfony/polyfill-ctype (v1.14.0): Downloading (100%)
+  - Installing phpoption/phpoption (1.7.2): Downloading (100%)
+  - Installing vlucas/phpdotenv (v4.1.0): Downloading (100%)
+  - Installing roave/security-advisories (dev-master 0365bf2)
+  - Installing squizlabs/php_codesniffer (3.5.4): Downloading (100%)
+ Generating optimized autoload files
+ [isabell@stardust isabell]$
+
+
+Configuration
+=============
+
+Wordpress-Configuration is done using .env-Files. Edit ``/var/www/virtual/$USER/bedrock/.env``
+
+In here you need to enter your :manual_anchor:`MySQL credentials <database-mysql.html#login-credentials>` database connection parameters. 
+We suggest using an :manual_anchor:`additional <database-mysql.html#additional-databases>` database. For example: ``isabell_wp``.
+
+.. code-block:: ini
+ :emphasize-lines: 1,2,3,10,14,15,18,19,20,21,22,23,24,25
+
+ DB_NAME='isabell_wp'
+ DB_USER='isabell'
+ DB_PASSWORD='MySuperSecretPassword'
+
+ # Optionally, you can use a data source name (DSN)
+ # When using a DSN, you can remove the DB_NAME, DB_USER, DB_PASSWORD, and DB_HO$
+ # DATABASE_URL='mysql://database_user:database_password@database_host:database_$
+
+ # Optional variables
+ DB_HOST='localhost'
+ # DB_PREFIX='wp_'
+
+ WP_ENV='development'
+ WP_HOME='isabell.uber.space'
+ WP_SITEURL="${WP_HOME}/wp"
+
+ # Generate your keys here: https://roots.io/salts.html
+ AUTH_KEY='generateme'
+ SECURE_AUTH_KEY='generateme'
+ LOGGED_IN_KEY='generateme'
+ NONCE_KEY='generateme'
+ AUTH_SALT='generateme'
+ SECURE_AUTH_SALT='generateme'
+ LOGGED_IN_SALT='generateme'
+ NONCE_SALT='generateme'
+
+You can use Roots' `Salt-Creator <https://roots.io/salts.html>`_ to generate the Salts.
+
+.. note:: You can leave the ``WP_ENV`` Setting on ``development`` for now, but don't forget to set it on ``production`` when launching your site.
+  See the Bedrock-Documentation_ for more Info on Environment-Settings
+
+You now need to set your :manual:`document root <web-documentroot>` to the ``bedrock/web/`` directory. To do this, we delete the standard document root folder and create a symlink instead.
+
+.. warning:: Please make sure your DocumentRoot is empty before removing it. This step will delete all contained files.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ rm -r /var/www/virtual/$USER/html
+ [isabell@stardust ~]$ ln -s /var/www/virtual/$USER/bedrock/web /var/www/virtual/$USER/html
+ [isabell@stardust ~]$
+
+
+Point your browser to your domain, ``https://isabell.uber.space/wp/wp-admin`` in this example, to start the wordpress installation process. Here you will

```

Comment:
This fails because wordpress doesn't want to create the DB itself. Please add a step in "Prerequisites" creating it like so:

```
$ mysql -e "CREATE DATABASE ${USER}_wp"
```

Reply:
ok, changed it and also the wording for the .env-Configuration to match.